src/components/app/App.jsx

```diff
@@ -45,6 +46,9 @@ export default function App() {
               <PageRoute exact path="/:enterpriseSlug/program/:programUuid" component={ProgramPage} />
             )}
             <PageRoute exact path="/:enterpriseSlug/program-progress/:programUUID" component={ProgramProgressPage} />
+            {features.ENABLE_PROGRAMS_PAGE && (
```

Comment:
I think we need to change the feature flag name and make it more descriptive. it is currently confusing with L46

Reply:
changed it to `ENABLE_PROGRAM_PROGRE_PAGE`